src/main/charts/bitbucket/templates/_helpers.tpl

```diff
@@ -524,6 +524,57 @@ volumeClaimTemplates:
 {{ end }}
 {{ end }}
 
+{{- define "bitbucket.openSearchEnvVars" -}}
+{{- if and .Values.bitbucket.clustering.enabled (or .Values.opensearch.enabled .Values.opensearch.baseUrl) }}
+- name: SEARCH_ENABLED
```

Comment:
> Set 'false' to prevent OpenSearch (previously Elasticsearch) from starting in the container. This should be used if OpenSearch is running remotely, e.g. for if Bitbucket is running in a Data Center cluster. You may also use ELASTICSEARCH_ENABLED to set this property, however this is deprecated in favor of SEARCH_ENABLED

This confused me slightly :)

Reply:
It should be probably false also if number of replicas is > 1, no?